components/faunadb/actions/execute-graphql-query/execute-graphql-query.mjs

```diff
@@ -0,0 +1,24 @@
+import faunadb from "../../faunadb.app.mjs";
+
+export default {
+  key: "faunadb-execute-graphql-query",
+  name: "Execute GraphQL Query",
+  description: "Performs an arbitrary authorized GraphQL query",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    faunadb,
+    query: {
+      label: "Query",
+      description: "A GraphQL query to execute",
+      type: "string",
+    },
+  },
+  async run({ $ }) {
+    const response = await this.faunadb.executeGraphqlQuery(this.query);
```

Comment:
Since `executeGraphqlQuery` users `axios`, we need to pass in the `$` variable.
```suggestion
    const response = await this.faunadb.executeGraphqlQuery(this.query, $);
```